web/client/components/manager/rulesmanager/RuleAttributes.jsx

```diff
@@ -11,7 +11,6 @@ const React = require('react');
 const {Panel} = require('react-bootstrap');
 const _ = require('lodash');
 const Select = require('./RuleAttributeSelect');
-const {head} = require('lodash');
 
```

Comment:
I'm trying to reduce clutter. The lodash wrapper is used anyway so another require is redundant.